rest/route/host_agent.go

```diff
@@ -1228,6 +1228,12 @@ func (h *hostAgentEndTask) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONInternalErrorResponder(err)
 	}
 
+	if evergreen.IsGithubMergeQueueRequester(t.Requester) {
+		if err = model.HandleEndTaskForGithubMergeQueueTask(ctx, t, h.details.Status); err != nil {
+			return gimlet.MakeJSONInternalErrorResponder(err)
```

Comment:
nit: format error with message

Reply:
HandleEndTaskForGithubMergeQueueTask returns a stack, so I think it would be too much to return one here.